packages/react-charting/src/components/AreaChart/AreaChart.types.ts

```diff
@@ -59,6 +59,18 @@ export interface IAreaChartProps extends ICartesianChartProps {
    * Optimize area chart rendering for large data set.
    */
   optimizeLargeData?: boolean;
+
+  /**
+   * Prop to set the x axis title
+   * @default undefined
+   */
+  xAxisTitle?: string;
```

Comment:
These should be defined in CartesianChartProps only. That should cover all cartesian based charts.